### DISCUSSION TITLE
SMTP setting

### DISCUSSION THREAD

**Discussion by rigdoge** [OpnForm/OpnForm#504] [Q&A] [answered]
Hello, I love this soft, it's really awesome.
May I ask if I wanna an email notifacation, how can I set up?
Thanks

**Comment by hallerdavid** [comment]
i have the same problem. I filled out the .env file in /api/ with my Mail settings and added the "mail notification" with my "to-Mail" but get no mails. How can i debug this? @JhumanJ can you maybe help me / us out a little bit?

**Comment by JhumanJ** [comment, marked as answer]
Hi, please make sure you followed this guide: https://docs.opnform.com/configuration/email-setup
And please share the laravel logs
